portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,7 +30,22 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println(WELCOME_MESSAGE);
+    ArrayList<String> sampleComments = new ArrayList<String>();
+    sampleComments.add("I love this page!");
+    sampleComments.add("Wow, what a great site.");
+    sampleComments.add("This Alisha girl seems really cool :)");
+
+    // convert the arraylist to json
+    String json = convertToJsonUsingGson(sampleComments);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJsonUsingGson(ArrayList<String> comments) {
+    Gson gson = new Gson();
+    String json = gson.toJson(comments);
+    return json;
```

Comment:
nit: you can simplify this to just `return new Gson().toJson(comments);`

alternatively, you could make Gson an instance field to avoid creating it each time this method is called